packages/app/pages/course/[cid]/insights.tsx

```diff
@@ -110,10 +135,15 @@ function RenderInsight({
     <Card
       size={insight.size as CardSize}
       title={insight.displayName}
+      style={{
+        margin: "8px",
+        width: insight.size === "default" ? "600px" : "200px",
+        height: insight.size === "default" ? "600px" : "150px",
```

Comment:
Instead of setting pixel amounts would it be better to have the insights each take up a percentage of the screen?

Reply:
I think that'd be more sensible yeah